### PR TITLE
add share feature description to README.md

### PR DESCRIPTION
**Pull request by alex-krasn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,10 +188,26 @@ When you want to resume your project, you first need to create a connection to t
 
 Finally, go to the main page (house icon) and click Open Cloud Project. Then select the blob storage connection, and select your project's .proj file. The application will load all of the project's settings because it has the security token.
 
+##### Share a project  #####
+
+FoTT allow sharing of projects with somebody who use the tool too and share access to same Azure Blob Storage container where a project is located.
+In order to share project follow these steps:
+###### On the sending site:
```
Comment (stew-ro) [inline README.md:195]: should be "side"

File: README.md [hunk 2]
```diff
@@ -187,10 +188,26 @@ When you want to resume your project, you first need to create a connection to t
...
+
+1. Open the project you want to share in the tool. In the right top corner find and click "share" icon ![alt text](docs/images/share-icon.png "Share project menu icon").
+You should see a message pop-up saying that your share string been saved in your clipboard.
+2. Share the string in you clipboard via any convenient way for you.
+3. On the receiving side
```
Comment (stew-ro) [inline README.md:200]: Step 3 isn't a step?

File: README.md [hunk 3]
```diff
@@ -187,10 +188,26 @@ When you want to resume your project, you first need to create a connection to t
...
+
+###### On the sending site:
```
Comment (stew-ro) [inline README.md:202]: Should be "On the receiving side"

**Comment by alex-krasn** [comment]
got weir errors when merging...

**Pull request merged**